core/org.eclipse.cdt.core.tests/META-INF/MANIFEST.MF

```diff
@@ -45,7 +45,8 @@ Require-Bundle: org.eclipse.core.resources,
  org.eclipse.ltk.core.refactoring;bundle-version="3.4.0",
  org.hamcrest.core,
  org.hamcrest.library,
- com.google.gson;bundle-version="[2.8.6,3.0.0)"
+ com.google.gson;bundle-version="[2.8.6,3.0.0)",
+ org.eclipse.cdt.build.gcc.core
```

Comment:
I think this dependency is a problem - not sure core.tests should depend on something not in cdt core.